src/client/src/orangehrmCorporateDirectoryPlugin/components/SummaryCard.vue

```diff
@@ -0,0 +1,306 @@
+        >
+          <oxd-icon
+            v-show="showTelephoneClip"
+            name="clipboard-check"
+          ></oxd-icon>
+        </div>
+      </div>
+      <oxd-divider></oxd-divider>
+      <div
+        class="orangehrm-directory-card-hover"
+        @mouseleave="showEmailClip = false"
+        @mouseover="showEmailClip = true"
+      >
+        <div class="orangehrm-directory-card-hover-body">
+          <oxd-text type="toast-message">{{ $t('Work Email') }}</oxd-text>
+          <oxd-text type="toast-title">
+            {{ employeeInfoDetails[0].employeeWorkEmail }}</oxd-text
+          >
+        </div>
+        <div
+          class="orangehrm-directory-card-hover-body orangehrm-directory-card-icon"
+        >
+          <oxd-icon v-show="showEmailClip" name="clipboard-check"></oxd-icon>
+        </div>
+      </div>
+      <oxd-divider></oxd-divider>
+      <div class="orangehrm-directory-card-qrcode">
+        <img
+          class="orangehrm-directory-card-qrcode-img"
+          src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d0/QR_code_for_mobile_English_Wikipedia.svg/800px-QR_code_for_mobile_English_Wikipedia.svg.png"
+        />
+      </div>
+    </div>
+  </oxd-sheet>
+</template>
+
+<script>
+import Sheet from '@ohrm/oxd/core/components/Sheet/Sheet';
+import Icon from '@ohrm/oxd/core/components/Icon/Icon';
+import ProfilePicture from '@/orangehrmCorporateDirectoryPlugin/components/ProfilePicture';
+import {APIService} from '@/core/util/services/api.service';
+
+const employeeInfoDetailsDataNormalizer = data => {
+  return data.map(item => {
+    return {
+      employeeId: item.id,
+      employeeWorkTelephone: item.contactInfo?.workTelephone,
+      employeeWorkEmail: item.contactInfo?.workEmail,
+    };
+  });
+};
+
+export default {
+  name: 'SummaryCard',
+  components: {
+    'oxd-sheet': Sheet,
+    'oxd-icon': Icon,
+    'profile-picture': ProfilePicture,
+  },
+  props: {
+    id: {
+      type: Number,
+      required: true,
+    },
+    employeeName: {
+      type: String,
+      required: true,
+    },
+    employeeDesignation: {
+      type: String,
+      required: true,
+    },
+    employeeSubUnit: {
+      type: String,
+      default: '',
+    },
+    employeeLocation: {
+      type: String,
+      default: '',
+    },
+    showEmployeeDetails: {
+      type: Boolean,
+      default: false,
+    },
+  },
+
+  data() {
+    return {
+      employeeInfoDetails: [
+        {
+          employeeWorkTelephone: null,
+          employeeWorkEmail: null,
+        },
+      ],
+      showTelephoneClip: false,
+      showEmailClip: false,
+      toggle: false,
+    };
+  },
+
+  computed: {
+    changeShowDetailsStatus() {
+      if (this.showEmployeeDetails === true && this.toggle === true) {
+        this.showEmployeeDetailsFn(this.id);
+      }
+      return this.showEmployeeDetails === true ? true : false;
+    },
+  },
+
+  methods: {
+    showEmployeeDetailsFn(id) {
+      new APIService(
```

Comment:
wrong use of APIService. APIService should be initiated once and reused for successive API calls. check other pages for reference.

Reply:
Done